src/main/java/net/openhft/chronicle/core/annotation/ScopeConfined.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016-2020 chronicle.software
+ *
+ * https://chronicle.software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.openhft.chronicle.core.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Indicates that the value consumer, value mapper, structure etc. should not divulge the object outside
+ * the annotated scope. The meaning of "the object" should also be construed to include any and all descendant
```

Comment:
I had to look some of these up in a dictionary

Reply:
same, but i love it )